javascript/organizing-js/es6-modules.md

```diff
@@ -121,26 +121,47 @@ function component() {
 document.body.appendChild(component());
 ~~~
 
-Easy! Now, if you run `npx webpack` in your project directory your page should show our new function being used.
+Easy! Now, if you run `npx webpack` in your project directory, your page should show our new function being used.
 
-There are 2 different ways to use exports in your code: named exports and default exports. Which option you use depends on what you're exporting. As a general rule if you want to export multiple functions use named exports with this pattern:
+There are 2 different ways to use exports in your code: named exports and default exports. Which option you use depends on what you're exporting. As a general rule, if you want to export multiple functions, use named exports like so:
 
 ~~~javascript
-const functionOne = () => 'ONE'
-const functionTwo = () => 'TWO'
+// a file called myModule.js
+const functionOne = () => 'ONE';
+const functionTwo = () => 'TWO';
 
 export {
   functionOne,
   functionTwo
-}
+};
 ~~~
 
-and to import them:
+And to import them:
 
 ~~~javascript
-import {functionOne, functionTwo} from './myModule'
+// main js file in /src folder
+import {functionOne, functionTwo} from './myModule';
 ~~~
 
 Using this pattern gives you the freedom to only import the functions you need in the various files of your program. So it's perfectly fine to only import `functionOne` if that's the only one you need.
 
-The various import/export methods are best explained in the docs that we linked earlier - [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [export](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export).
+However, if you do only need to export and import a _single_ object - variable, class or function - from a module, you can use the `default` keyword like so: 
```

Comment:
With all prior conversation in mind, I am happy to keep this example in this PR, but I am firm on retaining Webpack’s wording of their package’s responsibilities/function. The grammar fixes, of course, are fine.